forem.flay.txt

```diff
@@ -0,0 +1,913 @@
+Total score (lower is better) = 14503
```

Comment:
not opposed to code metrics, but are we checking this in (or did this get committed accidentally?)

Reply:
🤦🏾‍♀️